tests/Unit/Evolution/Systems/RelativisticEuler/Valencia/BoundaryCorrections/Rusanov.py

```diff
@@ -0,0 +1,149 @@
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+import numpy as np
+
+import Evolution.Systems.RelativisticEuler.Valencia.TestFunctions as valencia
+
+
+def dg_package_data_tilde_d(
+    tilde_d, tilde_tau, tilde_s, flux_tilde_d, flux_tilde_tau, flux_tilde_s,
+    lapse, shift, spatial_metric, rest_mass_density, specific_internal_energy,
+    specific_enthalpy, spatial_velocity, normal_covector, normal_vector,
+    mesh_velocity, normal_dot_mesh_velocity, use_polytropic_fluid):
```

Comment:
you can abbreviate all of these unused args with `dg_package_data_tilde_d(tilde_d, *args, **kwargs):`

Reply:
This is new info to me, so thanks for the pointer! I think will stay as-is for consistency with elsewhere, though?

Reply:
Up to you, but I don't think we have to prioritize consistency here. Also this is already used in `FishboneMoncriefDisk.py`, `KomissarovShock.py` and `FirstOrderInternalPenalty.py` (disclaimer: I wrote 2/3 of those).